### PR TITLE
Stage and prod configurations for external modules

### PR DESCRIPTION
**Pull request by sselvara**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #

Test URLs:
- Before: https://main--airindia--hlxsites.hlx.page/in/en/book/special-offers/global-connectivity
- After: https://booking-env-update--airindia--hlxsites.hlx.page/in/en/book/special-offers/global-connectivity

### CODE REVIEW

**Review by sdmcraft — APPROVED** [review]
@sselvara , we'd need a readme of steps on how to update the booking widget.

File: scripts/utils/constants.js [hunk 1]
```diff
@@ -1,10 +1,22 @@
 const CMP_CONFIG = {
   chatbotPlaceholder: '<div class="airindiaChatBot"></div>',
-  chatbotScript: 'https://code.jquery.com/jquery-3.7.1.min.js, https://cbot-scripts-staging.airindia.com/ai-chatbot.js',
-  chatbotStyle: 'https://cbot-scripts-staging.airindia.com/ai-chatbot.css',
+  chatbotScript: {
+    stage: 'https://code.jquery.com/jquery-3.7.1.min.js, https://cbot-scripts-staging.airindia.com/ai-chatbot.js',
+    prod: 'https://code.jquery.com/jquery-3.7.1.min.js, https://cbot-scripts-staging.airindia.com/ai-chatbot.js',
```
Comment (sdmcraft) [inline scripts/utils/constants.js:5]: The prod scripts also have "staging" in the url??

Comment (sselvara) [reply scripts/utils/constants.js]: @sdmcraft  Updates the configuration

File: blocks/booking/booking-stage.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+.booking.search-flight .booking-form {
+    max-width: 100%;
+}
+
+.booking.search-flight .add-return-label {
+    border: none;
+}
+
+.booking.search-flight .pays {
+    padding-bottom: 1rem !important;
```
Comment (sdmcraft) [inline blocks/booking/booking-stage.css:11]: `!important` is an anti-pattern. Can we manage without it, like using more specific selectors?

**Review by sdmcraft — APPROVED** [review]

File: blocks/booking/booking-stage.css (hunk 2 above)
Comment (sselvara) [reply blocks/booking/booking-stage.css:11]: @sdmcraft , !important is used in the existing stylesheet, hence we need to add it here to fix.

**Pull request merged**